cli/src/main/kotlin/commands/RequirementsCommand.kt

```diff
@@ -116,7 +116,11 @@ class RequirementsCommand : CliktCommand(name = "requirements", help = "List the
                         try {
                             val actualVersion = tool.getVersion()
                             try {
-                                if (tool.getVersionRequirement().isSatisfiedBy(actualVersion)) {
+                                val isRequiredVersion = tool.getVersionRequirement().let {
```

Comment:
consider naming the var: isVersionRequirementSatisfied

Reply:
I actually did, but found it to be too long 😀